src/foam/u2/view/PopUpTitledView.js

```diff
@@ -0,0 +1,241 @@
+        margin: 0;
+        margin-top: 20px;
+        padding: 0;
+        overflow: hidden;
+      }
+
+      ^ .cancel-Button, .foam-u2-ActionView-back {
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        background-color: rgba(164, 179, 184, 0.1);
+        box-shadow: 0 0 1px 0 rgba(9, 54, 73, 0.8);
+        cursor: pointer;
+        text-align: center;
+        font-family: Roboto;
+        font-size: 14px;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        margin-top: 5px;
+        margin-left: 20px;
+        margin-bottom: 20px;
+        position: fixed;
+        color: #093649;
+      }
+
+      ^ .foam-u2-ActionView-back:hover {
+        background-color: rgba(164, 179, 184, 0.1);
+        background: rgba(164, 179, 184, 0.1);
+      }
+
+      ^ .input-container {
+        margin-top: 20px;
+      }
+
+      ^ .pDefault {
+        margin-bottom: 8px;
+        margin-top: 0;
+        margin-left: 20px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: 300;
+        letter-spacing: 0.2px;
+        text-align: left;
+        color: #093649;
+      }
+
+      ^ .foam-u2-ActionView-closeButton {
+        width: 24px;
+        height: 24px;
+        margin: 0;
+        margin-top: -10px;
+        margin-right: 20px;
+        cursor: pointer;
+        display: inline-block;
+        float: right;
+        border: none;
+        background: transparent;
+        outline: none;
+      }
+
+      ^ .foam-u2-ActionView-closeButton:hover {
+        outline: none;
+        border: none;
+        background: transparent;
+      }
+
+      ^ .summary-heading {
+        padding: 10px 74px 0px 20px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: bold;
+        letter-spacing: 0.2px;
+        text-align: left;
+        color: #093649;
+      }
+    */}
+    })
+  ],
+
+  methods: [
+    function initE() {
+    this.SUPER();
+    var self = this;
+
+    this
+      .addClass(this.myClass())
+      .start()
+        .start().addClass('Message-Container')
+          .start().addClass('Change-Container')
+            .start().addClass('Change-Text').add(self.title).end()
+            .start()
+              .add(self.CLOSE_BUTTON)
+              .addClass('close-Button')
+            .end()
+          .end()
+          .add(self.messageView$.map(function(f) {
+            return foam.u2.ViewSpec.createView(f);
+          }))
+        .end()
+      .end()
+    }
+  ],
+
+  actions: [
+    {
+      name: 'closeButton',
+      icon: 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4gICAgPGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIiBzdHJva2U9IiNGRkYiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSIyIj4gICAgICAgIDxwYXRoIGQ9Ik01IDVsMTQuMTQyIDE0LjE0Mk0xOSA1TDUgMTkiLz4gICAgPC9nPjwvc3ZnPg==',
+      code: function (X) {
+        X.closeDialog();
```

Comment:
Is this needed? Can't it just close itself?

Reply:
From the docs, it seems like clicking the background closes itself. 

https://github.com/foam-framework/foam2/blob/ca06f9e4873d5bd4ee6a81d773f95114e97f5506/src/foam/u2/dialog/Popup.js#L23-L26

However this if for the `X` close button on the top right. Please let me know if there's another way.

Reply:
Who is exporting closeDialog and why do they need to? Can't the window just close itself without relying on some parent to do it? I think that this was meant to be more of a notification that the window was closed in case you're interested, like 'dialogClosed', but the dialog should just remove itself.